vvp-dossier/vvp-dossier.schema.json

```diff
@@ -0,0 +1,178 @@
+{
+  "$id": "EL3mANFohCJSfsZVc3jrqIKEZsLC-KdCC455UCOGPWGi",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Verifiable Voice Dossier Credential",
+  "description": "Attestation credential, issued by the accountable party, that assembles all of the evidence required as per the Verifiable Voice Protocol",
+  "type": "object",
+  "credentialType": "VvpDossierCredential",
+  "version": "1.0.0",
+  "required": [
+    "v",
+    "d",
+    "i",
+    "ri",
+    "s",
+    "a",
+    "e"
+  ],
+  "properties": {
+    "v": {
+      "description": "Version",
+      "type": "string"
+    },
+    "d": {
+      "description": "Credential SAID",
+      "type": "string"
+    },
+    "i": {
+      "description": "Issuer AID",
+      "type": "string"
+    },
+    "ri": {
+      "description": "Credential status registry",
+      "type": "string"
+    },
+    "s": {
+      "description": "Schema SAID",
+      "type": "string"
+    },
+    "a": {
+      "$id": "EEMVwHco23OmIGm4k3czzLZqePejuZH9Mr11kwNnMMX3",
+      "description": "Attributes block",
+      "type": "object",
+      "properties": {
+        "d": {
+          "description": "Attributes block SAID",
+          "type": "string"
+        },
+        "dt": {
+          "description": "Issuance datetime, ISO-8601 datetime string",
+          "format": "date-time",
+          "type": "string"
+        },
+        "LEI": {
+          "description": "LEI of the Accountavble Party",
+          "type": "string",
+          "format": "ISO 17442"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "d",
+        "dt"
+      ]
+    },
+    "e": {
+      "oneOf": [
+        {
+          "description": "Edges section SAID",
+          "type": "string"
+        },
+        {
+          "$id": "EBhxRTNOozLecDcwLmhcJeiIsaF2dP9Mcw0AFKSrbVsb",
+          "description": "Edges detail",
+          "type": "object",
+          "required": [
+            "d",
+            "alloc",
+            "tnalloc",
+            "delsig"
+          ],
+          "properties": {
+            "d": {
+              "description": "Edges section SAID",
+              "type": "string"
+            },
+            "vetting": {
+              "description": "Chain to a credential proves the identity of the accountable party as a legal entity",
+              "type": "object",
+              "required": [
+                "n",
+                "s"
+              ],
+              "properties": {
+                "n": {
+                  "description": "SAID of credential that proves the identity of the issuer",
+                  "type": "string"
+                },
+                "s": {
```

Comment:
I don't think we should constrain the edge so that it has to point to a vLEI. It should be any SAID, I think.

Reply:
removed LE schema constraint